src/speculators/data_generation/preprocessing.py

```
@@ -73,60 +73,60 @@ def _visualize_sample(preprocessed: HFDataset, processor: ProcessorLike, idx: in
-    """Normalize conversation to standard format with role/content keys.
-
-    Args:
-        conv: Raw conversation turns
-        turn_dropout: If True, randomly keeps first N consecutive turns (1 to len(conv))
-
-    Returns:
-        Normalized conversation with optional turn dropout applied
-    """
-    # Randomly pick how many consecutive turns to keep from the start
+    """Normalize conversation to standard format with role/content keys."""
     num_turns_to_keep = random.randint(1, len(conv)) if turn_dropout else len(conv)
```

Ruff (0.15.18)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)